rfc9000.md

```diff
@@ -1382,7 +1382,7 @@ when a client's address changes.
 
 A server in a deployment that does not implement a solution to maintain
 connection continuity when the client address changes SHOULD indicate that
-migration is not supported by using the disable_active_migration transport
```

Comment:
@janaiyengar explicitly added this "by" in 4601579.

Reply:
Yup -- @MikeBishop added "that" on line 1384, which I think makes this "by" necessary.